aws_policy_equivalence.go

```diff
@@ -383,6 +383,10 @@ func newAWSStringSet(members interface{}) awsStringSet {
 		}
 		actions := make([]string, len(multiple))
 		for i, action := range multiple {
+			if _, ok := action.(string); !ok {
+				return nil
```

Comment:
Return `nil` as this routine returns no `error` and I didn't want to touch all call sites.